components/compositing/compositor.rs

```diff
@@ -339,33 +341,31 @@ impl PipelineDetails {
     }
 }
 
-#[derive(Clone, Copy, Debug, PartialEq)]
-enum CompositeTarget {
-    /// Normal composition to a window
+#[derive(Clone, Debug, PartialEq)]
+pub enum CompositeTarget {
+    /// Draw directly to a window.
     Window,
 
-    /// Compose as normal, but also return a PNG of the composed output
-    WindowAndPng,
+    /// Draw to an offscreen OpenGL framebuffer object (IOCompositor::output_framebuffer_id).
```

Comment:
```suggestion
    /// Draw to an offscreen OpenGL framebuffer object [IOCompositor::output_framebuffer_id].
```
I believe `cargo doc` is able to create the link.

Reply:
done in 1f07d7bd76edf21bfb54842da7b96e14b6023e37